{"uid":"584aabde1aa01d38","name":"test_risk_total","fullName":"test_case.test_risk_overview.test_risk_total.TestCase#test_risk_total","historyId":"2c14c43617eab455f562a16ead07c8ac","time":{"start":1753963419071,"stop":1753963419071,"duration":0},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token","statusTrace":"cls = <class 'test_case.test_risk_overview.test_risk_total.TestCase'>\n\n    @classmethod\n    def setup_class(cls):\n        logger.info('========获取首页风险概览风险api总数数据，测试开始==========')\n        try:\n            with allure.step('【资产】-【api列表】接口'):\n                url = f'https://{sc_ip}/apione/v2/assets/list'\n                payload = {\"risk_level_ids\": [2, 3, 4], \"page_num\": 1, \"page_size\": 10,\n                           \"time_layout\": \"2006-01-02 15:04:05\",\n                           \"sort\": -1}\n                token = getattr(var, \"token\", None)\n                if not token:\n                    logger.error(\"token 获取失败\")\n>                   pytest.fail(\"setup_class: 未获取到 token\")\nE                   Failed: setup_class: 未获取到 token\n\ntest_case\\test_risk_overview\\test_risk_total.py:34: Failed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1753963414831,"stop":1753963415081,"duration":250},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"event_loop_policy","time":{"start":1753963415081,"stop":1753963415081,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"_xunit_setup_class_fixture_TestCase","time":{"start":1753963419073,"stop":1753963419079,"duration":6},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token\n","statusTrace":"  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\python.py\", line 792, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, cls)\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\python.py\", line 720, in _call_with_optional_argument\n    func(arg)\n  File \"D:\\pythonProject3\\autotest_api\\test_case\\test_risk_overview\\test_risk_total.py\", line 34, in setup_class\n    pytest.fail(\"setup_class: 未获取到 token\")\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n","steps":[{"name":"【资产】-【api列表】接口","time":{"start":1753963419075,"stop":1753963419075,"duration":0},"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token\n","statusTrace":"  File \"D:\\pythonProject3\\autotest_api\\test_case\\test_risk_overview\\test_risk_total.py\", line 34, in setup_class\n    pytest.fail(\"setup_class: 未获取到 token\")\n  File \"D:\\pythonProject3\\autotest_api\\.venv\\Lib\\site-packages\\_pytest\\outcomes.py\", line 177, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":1,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"testStage":{"status":"failed","statusMessage":"Failed: setup_class: 未获取到 token","statusTrace":"cls = <class 'test_case.test_risk_overview.test_risk_total.TestCase'>\n\n    @classmethod\n    def setup_class(cls):\n        logger.info('========获取首页风险概览风险api总数数据，测试开始==========')\n        try:\n            with allure.step('【资产】-【api列表】接口'):\n                url = f'https://{sc_ip}/apione/v2/assets/list'\n                payload = {\"risk_level_ids\": [2, 3, 4], \"page_num\": 1, \"page_size\": 10,\n                           \"time_layout\": \"2006-01-02 15:04:05\",\n                           \"sort\": -1}\n                token = getattr(var, \"token\", None)\n                if not token:\n                    logger.error(\"token 获取失败\")\n>                   pytest.fail(\"setup_class: 未获取到 token\")\nE                   Failed: setup_class: 未获取到 token\n\ntest_case\\test_risk_overview\\test_risk_total.py:34: Failed","steps":[],"attachments":[{"uid":"6ded3574021e2c4b","name":"log","source":"6ded3574021e2c4b.txt","type":"text/plain","size":219}],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":1},"afterStages":[],"labels":[{"name":"feature","value":"【首页】-【风险概览】-【风险api总数】"},{"name":"story","value":"【风险api总数】"},{"name":"severity","value":"normal"},{"name":"parentSuite","value":"test_case.test_risk_overview"},{"name":"suite","value":"test_risk_total"},{"name":"subSuite","value":"TestCase"},{"name":"host","value":"5CD2323XZM"},{"name":"thread","value":"34004-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_case.test_risk_overview.test_risk_total"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"584aabde1aa01d38.json","parameterValues":[]}